include/rmm_utils.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#define DEBUG_NO_RMM
+
+#include <sstream>
+#include <stdexcept>
+
+#define RMM_TRY_THROW( call )  if ((call)!=RMM_SUCCESS) \
+    {                                                   \
+      std::stringstream ss;                             \
+      ss << "ERROR: RMM runtime call  " << #call        \
+         << cudaGetErrorString(cudaGetLastError());     \
+      throw std::runtime_error(ss.str());               \
+    }
+
+#ifdef DEBUG_NO_RMM
```

Comment:
Most of this is redundant with functionality already in RMM. RMM is ready to use. Please use it and file RMM issues for problems you run into.